delta/plugins/storage/src/main/resources/storage.conf

```diff
@@ -90,16 +90,7 @@ storage {
   }
   files {
     # the files aggregate configuration
-    aggregate {
-      snapshot-strategy = ${app.defaults.snapshot-default}
-      stop-strategy = {
-        # duration since the last received message after which the aggregate actors should shut down.
-        lapsed-since-last-interaction = 1 minute
-      }
-      ask-timeout = 15 seconds
-      evaluation-max-duration = 3 seconds
-      stash-size = 10
-    }
+    aggregate = ${app.defaults.aggregate}
```

Comment:
not possible, since the default aggregate never passivates.
You can do:
```
    aggregate {
      snapshot-strategy = ${app.defaults.snapshot-default}
      stop-strategy {
          lapsed-since-last-interaction = 1 minute
      }
      ask-timeout = ${app.defaults.aggregate.ask-timeout}
      evaluation-max-duration = ${app.defaults.aggregate.evaluation-max-duration}
      stash-size = ${app.defaults.aggregate.stash-size}
    }
```